src/main/java/com/spotify/reaper/ReaperApplication.java

```diff
@@ -60,25 +65,43 @@
 public class ReaperApplication extends Application<ReaperApplicationConfiguration> {
 
   static final Logger LOG = LoggerFactory.getLogger(ReaperApplication.class);
-
+  public final static UUID reaperInstanceId = UUID.randomUUID();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)